client/src/actions/team.js

```diff
@@ -286,6 +285,24 @@ export const updateTeam = (teamId, teamData) => async (dispatch) => {
   }
 };
 
+/**
+ * This action creator will be called when a user want to delete their account
+ *
+ * @param {*} teamId id of the team
+ * @param {*} teamData data object of the data to be deleted
+ * @returns
+ */
+export const clearTeam = (teamId, isDelete) => async (dispatch) => {
+  try {
+    const body = {
+      isDeleteFlag: isDelete,
+    };
+    await api.clearTeam(teamId, body);
```

Comment:
Should there be a success dispatch after here?

Reply:
Also remember to merge main into your branch and deal with the conflicts in this file

Reply:
line 300 uses the successmessage generator which does the toast